src/app/src/test/kotlin/org/imperial/mrc/hint/unit/controllers/ErrorReportControllerTests.kt

```diff
@@ -1,18 +1,29 @@
 package org.imperial.mrc.hint.unit.controllers
 
-import com.nhaarman.mockito_kotlin.doReturn
-import com.nhaarman.mockito_kotlin.mock
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.github.kittinunf.fuel.core.Client
+import com.github.kittinunf.fuel.core.FuelManager
+import com.github.kittinunf.fuel.core.Response
+import com.github.kittinunf.fuel.core.requests.DefaultBody
+import com.nhaarman.mockito_kotlin.*
 import org.assertj.core.api.Assertions.assertThat
+import org.imperial.mrc.hint.ConfiguredAppProperties
 import org.imperial.mrc.hint.clients.FuelFlowClient
 import org.imperial.mrc.hint.controllers.ErrorReportController
+import org.imperial.mrc.hint.integration.SecureIntegrationTests
 import org.imperial.mrc.hint.models.ErrorReport
 import org.imperial.mrc.hint.models.Errors
 import org.junit.jupiter.api.Test
 import org.springframework.http.*
+import java.net.URL
 import java.time.Instant
 
-class ErrorReportControllerTests
+class ErrorReportControllerTests: SecureIntegrationTests()
```

Comment:
This class doesn't contain integration tests, so I don't think this is an appropriate superclass. If you're just wanting to use `readPropsFromTempFile()` then best to move that function to somewhere like existing `org.imperial.mrc.hint.helpers.Helpers`. Or make `can serialize request body` an actual integration test (see below).